src/components/Security/Users/CreateOrUpdate.vue

```diff
@@ -185,7 +185,12 @@ export default {
       this.$v.kuid.$model = value
     },
     onCredentialsChanged(payload) {
-      this.credentials[payload.strategy] = { ...payload.credentials }
+      const notEmptyFields = Object.keys(payload.credentials).filter(c => payload.credentials[c] !== "").length > 0
```

Comment:
I'd rather do this check right before submitting, don't you agree?

Reply:
I think he was trying to centralize the logic.

Reply:
Yes @Leodau
I can replace that on submit

Reply:
🤔 But... `onCredentialsChanged` is called on submit?

Reply:
Nope, `onCredentialsChanged` is called every time credentials input changed
I replaced it on submit